### PR TITLE
Open external links in new tab

### PR DESCRIPTION
**Pull request by dem4ron**
We probably want this. :)

### CODE REVIEW

File: app/views/community/show.html.haml [hunk 1]
```diff
@@ -10,7 +10,7 @@
             Accelerate your learning with Exercism&apos;s community.
           Ask questions, explore ideas, and get to know millions of other developers upskilling on Exercism.
 
-        = link_to 'https://forum.exercism.org/', class: "btn-primary btn-l mb-20 w-[fit-content]" do
+        = link_to 'https://forum.exercism.org/', target: "_blank", rel: "noreferrer", class: "btn-primary btn-l mb-20 w-[fit-content]" do
```
Comment (dem4ron) [inline app/views/community/show.html.haml]: I don't know if we are worried about omitting `Referer`_(sic!)_ header, but as a best practice I did it like this.

File: app/views/community/stories/show.html.haml [hunk 2]
```diff
@@ -14,7 +14,7 @@
             %h1.text-h1.mb-12.text-white= @story.title
             %p.text-p-large.mb-28.theme-dark= @story.blurb
             .flex.items-center
-              = link_to @story.youtube_external_url, class: 'youtube-link mr-32' do
+              = link_to @story.youtube_external_url, target: "_blank", rel: "noreferrer", class: 'youtube-link mr-32' do
```
Comment (iHiD) [inline app/views/community/stories/show.html.haml:17]: Keep this one.

**Comment by iHiD** [comment]
I actually want the forum links to stay inside the website and not be external links, so I think all but one (which I've commented on) should stay inside. I'm 50/50 on the Swag one.

File: app/views/community/show.html.haml [hunk 3]
```diff
@@ -85,7 +85,7 @@
       %h2.text-h2.text-white.text-unnamed15.mb-8 Explore our swag
       .leading-150.text-20.text-borderLight.mb-24
         Enhance your desk. Enhance your look.
-      = link_to "https://swag.exercism.org", class: 'btn-primary btn-l' do
+      = link_to "https://swag.exercism.org", target: "_blank", rel: "noreferrer", class: 'btn-primary btn-l' do
```
Comment (iHiD) [inline app/views/community/show.html.haml]:
```suggestion
      = link_to "https://swag.exercism.org", class: 'btn-primary btn-l' do
```

**Pull request merged**